### ISSUE TITLE
Well I'm excited

### ISSUE THREAD

**Issue by HypnoPants** [WiseOwl5/EnviroMineRevived#1]
I've been disappointed in a lot of old mods that died out, but none more than Enviromine... you've got me really excited about this!

**Comment by RoboMedZi** [comment]
It died :(

**Comment by Funwayguy** [comment]
Quick CPR!

We are all just really busy with school/university at the moment.

**Comment by WiseOwl5** [comment]
Yes, but don't worry. I haven't abandoned it! I've still been working on it where I can. And like I've said on another GitHub post summer is coming in 7-8 weeks and next year I will have a lighter course load.

**Comment by hyuntoe** [comment]
I'm waiting update, too. :)
